Feat (Feb release): Advanced map (#19877)
newrelic/docs-website: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx

* Due to their agentless nature, databases cannot have alerts set for them; only service calls made to the database are reported to New Relic.

## Advanced service maps [#about-advanced-maps]

<Callout title="preview">
  We're still working on this feature, but we'd love for you to try it out!

  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
</Callout>

While the existing service map offers valuable insights, it has certain limitations:
* **Fragmented experience**: Navigation across different views can lead to fragmented insights.
* **Incomplete map**: Limited ability to explore the entire cloud estate.
* **Inconsistency**: Discrepancies in data and interfaces across different maps.
* **Information overload**: Excessive data presentation can overwhelm users.
* **Missing entities and relationships**: Lack of automatic discovery of services and cloud resources.

> To address these limitations, we introduce the <DNT>**Advanced service map**</DNT>, which brings enhanced capabilities and a unified map experience. Learn more about [Advanced service map](/docs/new-relic-solutions/new-relic-one/ui-data/advanced-maps).

Reviewer: ```suggestion
To address these limitations, we have <DNT>**Advanced service map**</DNT>, which brings enhanced capabilities and a unified map experience. Learn more about [Advanced service map](/docs/new-relic-solutions/new-relic-one/ui-data/advanced-maps).
```
